test/Array/indexOf-test.js

```diff
@@ -0,0 +1,12 @@
+let expect = require ('chai').expect
+let _indexOf = require ('../src/Array/indexOf')
+
+describe('_indexOf()', () => {
+  it('Gets the index at which the first occurrence of value is found in array', () => {
+    expect(_indexOf([1, 2, 3, 4, 5], 5)).to.deep.equal(4)
```

Comment:
`deep.equal` only needed when comparing nested arrays or nested objects with each other, ala:
```js
expect( [ 1, 2, [ 3, 4 ], 5 ] ).to.deep.equal( [ 1, 2, [ 3, 4 ], 5 ] )
```
[Chai's documentation](http://chaijs.com/api/bdd/#method_deep) is really helpful and readable.